pkgs/by-name/ll/llama-cpp/package.nix

```diff
@@ -45,10 +45,7 @@ effectiveStdenv.mkDerivation (finalAttrs: {
 
   nativeBuildInputs = [ cmake ] ++ lib.optionals blasSupport [ pkg-config ] ++ lib.optionals cudaSupport [
     cudaPackages.cuda_nvcc
-
-    # TODO: Replace with autoAddDriverRunpath
-    # once https://github.com/NixOS/nixpkgs/pull/275241 has been merged
-    cudaPackages.autoAddOpenGLRunpathHook
```

Comment:
@yannham can you remove this old TODO?

Reply:
Oops, sorry, I thought I did.

Reply:
Oh, but I did, in fact :slightly_smiling_face: so it's good to go?

Reply:
My bad!